test/gpu/interop/cuBLAS.skipif

```diff
@@ -1,9 +1,16 @@
 #!/usr/bin/env python3
 
-from distutils.spawn import find_executable
 import os
 
-missing_nvcc = find_executable("nvcc") is None
+try:
+    # Module `distutils` is deprecated in Python 3.10 and will be removed in Python 3.12
+    # Prefer `shutil.which` in Python 3.2+
+    from shutil import which
+except ImportError:
+    # Backport for pre Python 3.2
+    from distutils.spawn import find_executable as which
```

Comment:
Anything in the `test/` directory requires python 3.7 or newer (from https://chapel-lang.org/docs/usingchapel/prereqs.html), so I think it'd be cleaner to just use `shutil`. (I'm happy to make this change in a followup PR if you're feeling short on time to make additional changes)